acid/features/history/tests/conftest.py

```diff
@@ -29,21 +46,23 @@ def _make_buildset(build_number, branch='master'):
                                 ref_url='http://acid.test/gitweb/',
                                 oldrev='', newrev='')
         commit()
-        make_build(build_number=build_number, buildset_id=buildset.id)
-        make_build(build_number=build_number, buildset_id=buildset.id)
+        for _each in range(0, number_of_builds):
```

Comment:
If you don't need to track index number you should use just underscore.
```python
for _ in range(0, number_of_builds):
```
you can read more at https://stackoverflow.com/a/5893946

------
And when you use `range` from 0 to n, you don't need to declare 0 as first argument.
These two snippets gives exactly the same result:
```python
for _ in range(0, number_of_builds):
```
```python
for _ in range(number_of_builds):
```